admin/vagrant-acceptance-targets/fedora-20/Vagrantfile

```diff
@@ -10,6 +10,329 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+clustercrt = <<EOL
```

Comment:
Additionally, this particular Vagrantfile is based off the tutorial one (` config.vm.box = "clusterhq/flocker-tutorial"`) so you don't need to duplicate the logic here (even if it was useful in testing).